lib/SetAPI/SetAPIImpl.py

```diff
@@ -322,9 +322,11 @@ def save_differential_expression_matrix_set_v1(self, ctx, params):
         # ctx is the context object
         # return variables are: result
         # BEGIN save_differential_expression_matrix_set_v1
-        ws = Workspace(self.workspaceURL, token=ctx["token"])
-        demsi = DifferentialExpressionMatrixSetInterfaceV1(ws)
-        result = demsi.save_differential_expression_matrix_set(ctx, params)
+        ws = Workspace(self.workspace_url, token=ctx["token"])
+        set_interface = SetInterfaceV1(ws)
+        result = set_interface.save_set(
+            DIFFERENTIAL_EXPRESSION_MATRIX, ctx["provenance"], params
+        )
```

Comment:
`save` requests go to the SetInterfaceV1 class with params, provenance, and a string denoting the type of set to save